tests/unit_tests/test_wave_02.py

```diff
@@ -55,3 +55,45 @@ def test_get_item_by_id_no_matching():
     assert item_a in items
     assert item_b in items
     assert item_c in items
+
+
+#additional tests
+def test_items_have_default_age():
+    item = Item()
+    assert item.age == 0
+
+def test_items_have_custom_age():
+    item = Item(age=5)
+    assert item.age == 5
+
+
+# Additional test for Item id attribute exception handling
+def test_id_exception():
+    with pytest.raises(TypeError):
+        Item(id ="aaa")
```

Comment:
I would opt to move these tests into another file, perhaps `Wave 02` test file since it is testing the `Item` class. You could possibly argue for a creating different file just for these tests  since the test files are set up for individual waves. However, I do think a common consensus would be to remove them from here.